identity_base_config/recipes/default.rb

```diff
@@ -64,11 +64,11 @@
   mode '0755'
 end
 
-cookbook_file '/usr/local/bin/id-git' do
```

Comment:
Don't think we mean to delete `id-git`

Reply:
That... is odd. Must have been a misfire; fixing.